ui/src/data-services/hooks/deployments/useCreateDeployment.ts

```diff
@@ -14,7 +13,7 @@ const convertToServerFieldValues = (fieldValues: DeploymentFieldValues) => ({
   latitude: fieldValues.latitude,
   longitude: fieldValues.longitude,
   occurrences: [],
-  project: null,
+  project: `http://api.dev.insectai.org/api/v2/projects/${fieldValues.projectId}/`, // TODO: Can we pass just id here?
```

Comment:
I'm getting 400 if I just pass the ID here @mihow

Reply:
I just fixed this! The deployments endpoint now accepts a project id for the project field, and it returns a nested representation of the project instead of just the details URL.

<img width="515" alt="image" src="https://github.com/RolnickLab/ami-platform/assets/158175/ceb03103-4982-4df9-b7b1-c9194af90bc6">

Reply:
Thank you! Just pushed a FE fix